pkg/repository/events.go

```diff
@@ -25,9 +26,16 @@ func NewTCPEventClientRepo(config types.FluentBitEventConfig) EventRepository {
 		log.Println("[WARNING] fluentbit host does not appear to be up, events will be dropped")
 	}
 
+	// Parse event mapping
+	eventTagMap := make(map[string]string)
```

Comment:
you might not need the mapping if you use specific event types right?

Reply:
The mapping is generic to easily map any event to different outputs

Reply:
That means users will be able to better direct where they want events to go. E.g. it goes to an API, elastic search, kinesis, or straight to slack through fluentbit